src/sentry/integrations/slack/message_builder/notifications/base.py

```diff
@@ -65,7 +65,10 @@ def build(self) -> SlackAttachment | SlackBlock:
 
         actions_block = []
         for action in actions:
-            if action.label == "Turn on personal notifications" or action.url:
+            if (
+                action.label in ["Turn on personal notifications", "Approve", "Reject"]
```

Comment:
Do we know all the labels? Are they logged anywhere to see what else we're possibly missing?

Reply:
we can check all the possible instances where we create `MessageAction` objects

Reply:
ah okay. this logic is only used after we call `get_message_actions` on a notification, so we'd just have to double check what `MessageActions` we're creating through the notification platform.

Reply:
ok just checked, every other `MessageAction` object has a URL or has "Turn on personal notifications" as a label so we just missed 2 of the possible ones without URLs.